src/components/common/FormBottomBar.tsx

```diff
@@ -60,6 +60,7 @@ export default function FormBottomBar({
       className={classes.bottomBar}
     >
       <Button
+        id='cancelBottomBar'
```

Comment:
This component is used in all those full page forms that have a save/cancel at the bottom of the page. 
We could maybe add more context to the component - in the form of a prop, just for the id purposes.
Current change is totally fine for now.

Reply:
I had done what you suggested originally, but changed it back before committing.  You convinced me though - I will change it back.